cmd/checkpoint/main.go

```diff
@@ -295,6 +351,11 @@ func writeCheckpointManifest(pod *v1.Pod) error {
 	return writeAndAtomicRename(path, b, 0644)
 }
 
+// isPodCheckpointer returns true if the pod is the checkpointer itself.
+func isPodCheckpointer(pod *v1.Pod) bool {
+	return strings.HasPrefix(pod.Name, "pod-checkpointer")
```

Comment:
Yeah. We could probably check that maybe:

```
pod.Name == self.Namespace
pod.Name == self.Name + "-" + self.NodeName // Because we care about the static pod, right?
```